src/DetectChanges/Variance/TypeWithReflectorScope.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\DetectChanges\Variance;
+
+use Roave\BetterReflection\Reflection\ReflectionIntersectionType;
+use Roave\BetterReflection\Reflection\ReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionUnionType;
+use Roave\BetterReflection\Reflector\Reflector;
+
+/**
+ * This minimal value object carries an {@see ReflectionType} and the {@see Reflector}
```

Comment:
This entire file is no longer needed after https://github.com/Roave/BetterReflection/pull/836